### PR TITLE
refactor stream cmd

### PR DESCRIPTION
**Pull request by calvinmvrk**
Refactor command tool to except -fill [filename] input

Screenshot 2024-11-19 at 15 58 52

### CODE REVIEW

File: cmd/stream.go [hunk 1]
```diff
@@ -15,20 +15,25 @@ const (
...
-	Long:  `The stream command allows you to initiate a data stream to your agent.`,
+	Use:   "stream --file [filename]",
+	Short: "Parse stream data from agent",
+	Long:  `Allows you to parse a data stream to your agent response.`,
 	Run:   agentStream,
 }
 
 func init() {
 	streamCmd.PersistentFlags().String(streamCmdFileFlag, "", "Parse agent responses from a file")
+	rootCmd.AddCommand(streamCmd)
```
Comment (theodoracheng) [inline cmd/stream.go:26]: I think this was already added in the rootcmd file, so no need to add it here

**Review by theodoracheng — COMMENTED** [review]
Nice refactor! Just one minor comment

File: cmd/stream.go (hunk 1 above)
Comment (calvinmvrk) [reply cmd/stream.go]: updated! idk why i thought this was a breaking change at one point

Comment (theodoracheng) [reply cmd/stream.go]: will this still be needed in the future? or would it be ok to remove it completely?

**Review by theodoracheng — APPROVED** [review]

**Pull request merged**